azure-servicebus/src/main/java/com/microsoft/azure/servicebus/IMessageReceiver.java

```diff
@@ -329,7 +336,20 @@
      */
     Instant renewMessageLock(IMessage message) throws InterruptedException, ServiceBusException;
 
-    //Collection<Instant> renewMessageLockBatch(Collection<? extends IBrokeredMessage> messages) throws InterruptedException, ServiceBusException;    
+    /**
+     * Renews the lock on the message specified by the lock token. The lock will be renewed based on the setting specified on the entity.
+     * When a message is received in {@link ReceiveMode#PEEKLOCK} mode, the message is locked on the server for this
+     * receiver instance for a duration as specified during the Queue/Subscription creation (LockDuration).
+     * If processing of the message requires longer than this duration, the lock needs to be renewed. For each renewal, the lock is renewed by
+     * the entity's LockDuration.
```

Comment:
Can you write this for the async method and write here that sync method is just a sync version of the async method. Async methods are first class, sync methods are just wrappers on top of async methods.